tox.ini

```diff
@@ -7,5 +7,5 @@ deps =
     mock
     coverage>=4.0a5
     unittest2
-    git+http://github.com/Metaswitch/python-etcd.git@d5e8dc849f200ecfcb207086ce0d44f238c82aa5#egg=python-etcd
```

Comment:
The tag is great, but doesn't solve the packaged side of the problem. Gotta bump the version number too. ;)